services/user-svc/internal/organization/organization.go

```diff
@@ -487,16 +517,18 @@ func GetOrganizationById(db *gorm.DB, id uuid.UUID) (*Organization, error) {
 	return &organization, nil
 }
 
-func GetInvitationById(db *gorm.DB, id uuid.UUID) (*Invitation, error) {
```

Comment:
Please leave the method as it is. Just add the new one.

Reply:
But then we have an unused Funktion

Reply:
Yes. You are right. But this function can be used later.

Reply:
OK